keylime-agent/src/common.rs

```diff
@@ -83,32 +78,6 @@ where
     }
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize)]
-pub struct AuthTag {
-    bytes: Vec<u8>,
-}
-
-impl AsRef<[u8]> for AuthTag {
-    fn as_ref(&self) -> &[u8] {
-        self.bytes.as_slice()
-    }
-}
-
-impl TryFrom<&[u8]> for AuthTag {
-    type Error = String;
-
-    fn try_from(v: &[u8]) -> std::result::Result<Self, Self::Error> {
-        match v.len() {
-            AUTH_TAG_LEN => {
-                Ok(AuthTag { bytes: v.to_vec() })
-            }
-            other => Err(format!(
-                "auth tag length {other} does not correspond to valid SHA-384 HMAC",
-            )),
-        }
-    }
-}
-
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
 pub struct EncryptedData {
```

Comment:
@ansasaki : Should not this EncryptedData be removed from here? Aren't we duplicating it?

Reply:
Yes, you are right, we left it behind. Thanks for noticing!